examples/fluent-bit/grafana-loki/README.md

```diff
@@ -0,0 +1,23 @@
+### FireLens Example: Logging to Grafana Loki with Fluent Bit
+
+For documentation on sending your FireLens monitored log data to Grafana Loki, see: [Fluent Bit and Firelens in the Grafana documentation] (https://grafana.com/docs/loki/latest/clients/fluentbit/). The Loki data source provides access to Loki, Grafana’s log aggregation system.
+
+It should be noted that this example and Grafana docs show `"enable-ecs-log-metadata":"true"` (which is the default). This option tells FireLens to add ECS Task Metadata keys to logs.  
+
+For all configuration parameters for Fluent Bit Grafana output plugin, see [Grafana](https://grafana.com/docs/loki/latest/clients/fluentbit/#configuration-options) or [Fluent Bit](https://docs.fluentbit.io/manual/v/1.9-pre/pipeline/outputs/loki) documentation.
```

Comment:
THis links to 1.9 docs, why not link to latest?

Reply:
Aren't we using fluent bit 1.9 in our official images?